### PR TITLE
[Mint] Feature enforcement

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/mint-components/package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "workbox-build": "4.3.1"
   },
   "dependencies": {
-    "@raisins/stencil-docs-target": "^1.0.0",
+    "@raisins/stencil-docs-target": "^1.0.2-3",
```
Comment (noahwc) [inline packages/mint-components/package.json:70]: Move to a stable version

File: packages/mint-components/src/components/sqm-leaderboard/Leaderboard.stories.tsx [hunk 2]
```diff
@@ -163,6 +163,10 @@ const defaultStyles = {
   anonymousUser: "Anonymous User",
 };
 
+const link = <a>Support</a>;
+
+const tag = "Contact" + link + "about upgrading your plan";
```
Comment (noahwc) [inline packages/mint-components/src/components/sqm-leaderboard/Leaderboard.stories.tsx:168]: Needs a real link, should be a template string for proper spacing between link and text

File: packages/mint-components/src/components/sqm-leaderboard/sqm-leaderboard-view.tsx [hunk 3]
```diff
@@ -114,6 +118,7 @@ export function LeaderboardView(props: LeaderboardViewProps) {
         {styleString}
         {vanillaStyle}
       </style>
+      <div>Leaderboards</div>
```
Comment (noahwc) [inline packages/mint-components/src/components/sqm-leaderboard/sqm-leaderboard-view.tsx:121]: Does this header have the right styling?

File: packages/mint-components/src/components/sqm-rewards-table/RewardsTableCell.stories.tsx [hunk 4]
```diff
@@ -346,6 +353,18 @@ export const SourceCellDeletedUser = () => {
   );
 };
 
+export const SourceCellDeletedReferral = () => {
+  return (
+    // AL: TODO
```
Comment (noahwc) [inline packages/mint-components/src/components/sqm-rewards-table/RewardsTableCell.stories.tsx:358]: Flagging TODO

**Review by noahwc — COMMENTED** [review]
Just want to double check that all the new UI strings have gone through copy review too

File: packages/vanilla-components/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/vanilla-components",
   "title": "Vanilla Components",
-  "version": "1.2.8",
+  "version": "1.2.9-4",
```
Comment (00salmon) [inline packages/vanilla-components/package.json:4]: revert these changes - vanilla-components are updated in a different branch for feature enforcement

File: packages/mint-components/src/components/sqm-popup-container/sqm-popup-container.tsx [hunk 6]
```diff
@@ -15,9 +15,9 @@ import { getProps } from "../../utils/utils";
 })
 export class PopupContainer {
   /**
-   * Show Powered by Impact.com link
+   * Hide Powered by Impact.com link
```
Comment (00salmon) [inline packages/mint-components/src/components/sqm-popup-container/sqm-popup-container.tsx:18]:
```suggestion
   * Show Powered by Impact.com link
```

File: packages/mint-components/src/components/sqm-popup-container/sqm-popup-container.tsx [hunk 7]
```diff
@@ -15,9 +15,9 @@ import { getProps } from "../../utils/utils";
...
    *
-   * @uiName Show powered by
+   * @uiName Hide powered by
```
Comment (00salmon) [inline packages/mint-components/src/components/sqm-popup-container/sqm-popup-container.tsx:20]: The label was reversed without any of the logic being reversed, this is causing it to show up when "Hide Powered By" is true
it needs to stay as "Show Powered By""
(same problem in vanilla-components PR)

**Review by Locrian24 — CHANGES_REQUESTED** [review]
Just needs official `@raisins/stencil-docs-target` version once it's published

File: packages/mint-components/package.json (hunk 1 above)
Comment (Locrian24) [reply packages/mint-components/package.json]: flagges for release

**Review by Locrian24 — DISMISSED** [review]
Ready for deploy

**Review by Locrian24 — APPROVED** [review]

**Review by Locrian24 — DISMISSED** [review]

**Review by Locrian24 — APPROVED** [review]

**Pull request merged**